### PR TITLE
Delete dup docs

### PR DESCRIPTION
**Pull request by SSuranie**

### CODE REVIEW

File: docs/developers/contributing.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
+Documentation for the project is stored in the [docs](https://github.com/prebid/prebid.github.io/tree/master/prebid-server) directory. If your feature requires docs updates,
```
Comment (hhhjort) [inline docs/developers/contributing.md]: I think we should also point to the prebid.org URL that has the final, pretty version of the docs. Keep this link so people know where to go if anything needs to be added or edited.

**Comment by MartianTribe** [comment]
made changes per Hans comments.

File: docs/developers/contributing.md [hunk 2]
```diff
@@ -29,12 +29,14 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
-those updates must be submitted in the same Pull Request as the code changes.
+Documentation for the project is stored in the prebid-server directory on the [Prebid repo](https://github.com/prebid/prebid.github.io/tree/master/prebid-server). If your feature requires docs updates,
+those updates must be submitted in the same Pull Request as the code changes. Documentation can be viewed at
```
Comment (mansinahar) [inline docs/developers/contributing.md]: Shouldn't this: `If your feature requires docs updates,
those updates must be submitted in the same Pull Request as the code changes` sentence be updated to say that the doc updates need to be submitted in a _separate_ PR on the [prebid.github.io](https://github.com/prebid/prebid.github.io) repo?

File: docs/developers/contributing.md [hunk 3]
```diff
@@ -29,12 +29,14 @@ Bugfixes should include a regression test which prevents that bug from being re-
 
 ## Update Documentation
 
-Documentation for the project is stored in the [docs](..) directory. If your feature requires docs updates,
-those updates must be submitted in the same Pull Request as the code changes.
+Documentation for the project is stored in the prebid-server directory on the [Prebid repo](https://github.com/prebid/prebid.github.io/tree/master/prebid-server). If your feature requires docs updates,
+those updates must be submitted in the same Pull Request as the code changes. Documentation can be viewed at 
 
 - [docs/endpoints](../endpoints) describes the Prebid Server API. For example, the endpoint `host:port/openrtb2/auction` is described by [docs/endpoints/openrtb2/auction.md](../endpoints/openrtb2/auction.md)
```
Comment (mansinahar) [inline docs/developers/contributing.md]: The links here should to be updated to their appropriate location in the [prebid.github.io](https://github.com/prebid/prebid.github.io) repo.

**Comment by hhhjort** [comment]
We still have a .DS_Store file in the PR, and it now has some merge conflicts.

**Comment by hhhjort** [comment]
Are you planning on fixing this Steve?

**Comment by bretg** [comment]
Closing this per request from @SSuranie

**Comment by SSuranie** [comment]
Hi Hans

Sorry for letting this PR go. I had Bret delete it. I had some weird access issues when  I created it and for some reason this is the only git push I’ve done that has pulled the .ds_store file. Maybe I turned my invisibles on for some reason and then pushed. Anyway, I will create another branch, delete the appropriate files and submit a new PR. Again, sorry for not getting to this.

Steve

**Pull request closed without merging**